### DISCUSSION TITLE
Support For Tekton PipelineRuns

### DISCUSSION THREAD

**Discussion by pramodbindal** [kubernetes-sigs/kueue#3608] [General]
Hi,
I found this project interesting.
I was wondering if  we can utilize Kueue for PipelineRun scheduling.

so if cluster does not enough resources  then we can make new pipelines to wait until older ones are completed

**Comment by mimowo** [comment]
cc @mwielgus

**Comment by tenzen-y** [comment]
You might be interested in this issue: https://github.com/kubernetes-sigs/kueue/issues/74
